services/editor/src/components/common/Input/TypedInput.js

```diff
@@ -8,6 +8,9 @@ import { showCustomAlert, buttons } from '../../../store/ducks/alerts';
 import './TypedInput.css';
 import { connect } from 'react-redux';
 import { withJsonEditor } from './EditJSON';
+import * as R from 'ramda';
+import { WithContext as ReactTags } from 'react-tag-input';
+import * as TypesService from '../../../services/types-service';
```

Comment:
remove dependency to typesService, can be used with "types" prop that we get from context.